private4/index.php

```diff
@@ -43,7 +43,7 @@ function updateForHash() {
   <input type="text" style="-webkit-font-smoothing:antialiased;-webkit-transition:color .2s ease-in-out;-moz-transition:color .2s ease-in-out;-o-transition:color .2s ease-in-out;-ms-transition:color .2s ease-in-out;transition:color .2s ease-in-out;font-family: Lucida Blackletter; text-docoration:none; font-weight:bold; font-size: 34px; text-align:left; text-shadow: 5px 5px 10px #191919;color: #FFFFFF; background-color:#666666; margin-left:10px;border:none";id="my_text-input3" value="Genealogisk forum">
   <br>
   <div class="flex-container">
-<div class="flex-item"><input type="text" style="font-family: arial; font-size: 12px; font-weight:normal; text-align:center; color: #e1e1e1; background-color:#666666; width: 300px; margin-left:0px;border: 1px solid #f6f6f6; ";id="my_text-input3" value="Test bog "></div> 
+<div class="flex-item"><input type="text" style="font-family: arial; font-size: 12px; font-weight:normal; text-align:center; color: #e1e1e1; background-color:#666666; width: 300px; margin-left:0px;border: 1px solid #f6f6f6; ";id="my_text-input3" value="Test bog "></div>
```

Comment:
My editor automatically removes trailing spaces from lines... sorry if that creates extra noise here. You can just ignore changes that are just spaces like this.